recipes/nose2/meta.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: nose2
+  version: "0.7.4"
+
+source:
+  fn: nose2-0.7.4.tar.gz
+  url: https://files.pythonhosted.org/packages/93/46/a389a65237d0520bb4a98fc174fdf6568ad9dcc79b9c1d1f30afc6776031/nose2-0.7.4.tar.gz
+  md5: 4eea4cf00f0e0ceba686173774f15c2d
+
+build:
+  number: 0
+  skip: True  # [win]
+  entry_points:
+    - nose2 = nose2:discover
+    - nose2-3.6 = nose2:discover
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six >=1.7
+    - coverage >=4.4.1
```

Comment:
Is this a build dep?

Reply:
Yes, it was added by "conda-skeleton".

Reply:
Just because it was added by conda skeleton don't mean it is a build dependency.

Reply:
That is fixed now, thanks for the feedback!